.jenkins/common.groovy

```diff
@@ -21,7 +21,7 @@ def runCompileCommand(platform, project, boolean sameOrg=false)
                 set -x
                 cd ${project.paths.project_build_prefix}
                 ${getDependenciesCommand}
-                export PATH=/opt/rocm/hsa/include:$PATH
+                export PATH=/opt/rocm/hsa/include:\$PATH
```

Comment:
$$ doesn't work?

Reply:
I don't think $$ works in this kind of string. https://groovy-lang.org/syntax.html#_string_summary_table

Reply:
would have been nice ...